core/services/registrysyncer/orm.go

```diff
@@ -131,20 +131,34 @@ func NewORM(ds sqlutil.DataSource, lggr logger.Logger) orm {
 }
 
 func (orm orm) AddLocalRegistry(ctx context.Context, localRegistry LocalRegistry) error {
+	orm.lggr.Debugw("Adding local registry to DB...")
 	return sqlutil.TransactDataSource(ctx, orm.ds, nil, func(tx sqlutil.DataSource) error {
 		localRegistryJSON, err := localRegistry.MarshalJSON()
 		if err != nil {
 			return err
 		}
 		hash := sha256.Sum256(localRegistryJSON)
-		_, err = tx.ExecContext(
+		r, err := tx.ExecContext(
 			ctx,
-			`INSERT INTO registry_syncer_states (data, data_hash) VALUES ($1, $2) ON CONFLICT (data_hash) DO NOTHING`,
+			`INSERT INTO registry_syncer_states (data, data_hash) 
```

Comment:
Can you add a comment explaining that we insert only if the latest one is not identical? It took me a moment to grok this query.